### ISSUE TITLE
Gnome tweak tool boomed

### ISSUE THREAD

**Issue by bobcao3** [AOSC-Dev/aosc-os-abbs#4]
It has a bug so I can't switch my themes and fonts .... orz

**Comment by MingcongBai** [comment]
```
aosc [ aosc-os-abbs@master ] $ gnome-tweak-tool 
/usr/lib/python2.7/site-packages/gi/overrides/__init__.py:306: Warning: The property GSettings:schema is deprecated and shouldn't be used anymore. It will be removed in a future version.
  return super_init_func(self, **new_kwargs)
CRITICAL: Could not find schema /home/rmatos/Source/git.gnome.org/install/share/glib-2.0/schemas/org.gnome.shell.gschema.xml
WARNING : Shell not installed or running
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/gtweak/app.py", line 47, in do_startup
    assert(os.path.exists(gtweak.PKG_DATA_DIR))   
AssertionError
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/gtweak/app.py", line 38, in do_activate
    model = TweakModel()            
  File "/usr/lib/python2.7/site-packages/gtweak/tweakmodel.py", line 95, in __init__
    assert(os.path.exists(self._tweak_dir))
AssertionError
```

I will attempt a version bump.

**Comment by MingcongBai** [comment]
Fixed as of 8ba84a8bc650f6140178e7b04b6bdb2dd06debf4.
